contentcuration/contentcuration/static/js/edit_channel/exercise_creation/views.js

```diff
@@ -14,8 +14,8 @@ require("exercises.less");
 require("quilljs/dist/quill.snow.css");
 require("dropzone/dist/dropzone.css");
 
-var placeholder_text = "$1\${☣ LOCALPATH}/$3"
-var regExp = /\(\${☣ LOCALPATH}\/([^)]+)\)/g;
+var placeholder_text = "$1\${☣ CONTENTSTORAGE}/$3"
+var regExp = /\${☣ CONTENTSTORAGE}\/([^)]+)/g;
```

Comment:
Isn't this in `le-utils` already? I guess now we have to think of how to pass in data from `le-utils` to JS code.